src/python/platforms/fuchsia/device.py

```diff
@@ -16,6 +16,119 @@
 # TODO(mbarbella): Re-enable this check once functions below are implemented.
 # pylint: disable=unused-argument
 
+import os
+
+from fuchsia import errors
+from google_cloud_utils import gsutil
+from metrics import logs
+from system import environment
+from system import new_process
+from system import shell
+
+
+def qemu_setup():
+  """ Sets up and runs a QEMU VM in the background.
+  Does not block the calling process.
+  Fuchsia fuzzers assume a QEMU VM is running; call this routine prior to
+  beginning Fuchsia fuzzing tasks.
+  This initialization routine assumes that the GCS bucket contains the
+  standard Fuchsia SDK, as well as:
+  * /qemu-for-fuchsia/*
+  * /.ssh/* """
+  fuchsia_bucket_name = 'fuchsia_on_clusterfuzz_resources_v1'
```

Comment:
this seems to be only used for the local path, and isn't actually being used as a bucket name. just use the string directly later in the path.join, and maybe rename to somethng like 'fuchsia-resources'.